### DISCUSSION TITLE
RTX 4050 Laptop m.2

### DISCUSSION THREAD

**Discussion by wallentx** [geerlingguy/raspberry-pi-pcie-devices#645] [Ideas]
https://videocardz.com/newz/geforce-rtx-4050-laptop-gpu-add-in-board-spotted-with-m-2-connectors-instead-of-mxm

I'd love to get my hands on one of these if I could find a way to buy one without the laptop 👀

**Comment by geerlingguy** [comment]
I migrated this to a discussion, mostly because that's a very exotic (and non-standard) use of the M.2 slots. It looks like it takes a few and combines all the lanes for this card, and would require some weird contortions to try to replicate it with a Pi, at least involving a PCIe switch, and likely not being workable :(

**Reply by wallentx** [reply]
Dang, I misunderstood the image and thought the entire GPU was the m.2 NVME SSD. Yeah, this one is strange. I do have one of these [RTX A500](https://www.adlinktech.com/en/pocket-ai-with-nvidia-rtx-a500-egpu) egpus that requires Thunderbolt 3. Thunderbolt support is in the kernel for aarch64.. but I've not seen anything about anyone successfully setting up thunderbolt on a RPi. Likely to do with proprietary header pins on add-in cards.
